frontend/src/homePage/CodeGroups.jsx

```diff
@@ -0,0 +1,124 @@
+import { useState, useEffect } from "react";
+import "./CodeGroups.css";
+import axios from "axios";
+
+function CodeGroups() {
+  const [allGroups, setAllGroups] = useState([]);
+  const [groupName, setGroupName] = useState("");
+  const [members, setMembers] = useState([]);
+  const [memberInput, setMemberInput] = useState("");
+  const [error, setError] = useState("");
+  const [count, setCount] = useState(0);
+
+  const BACKEND_URL = import.meta.env.VITE_BACKEND_URL;
+
+  useEffect(() => {
+    setAllGroups([]);
+    const token = localStorage.getItem("jwt");
+    if (!token) {
+      return;
+    }
+
+    const getGroups = async () => {
+      const options = {
+        headers: {
+          Authorization: `Bearer ${token}`,
+        },
+        withCredentials: true,
+      };
+
+      try {
+        const createdGroups = await axios.get(
+          `${BACKEND_URL}/api/user/created-groups`,
+          options
+        );
+
+        const addedGroups = await axios.get(
```

Comment:
Small comment, but I would consider renaming this variable, I think `addedGroups` is somewhat vague and isn't immediately clear how it's different from `createdGroups`. Maybe something like `groupMemberships`

Reply:
Renamed. Thanks